.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: go
 sudo: false
 go:
-  - 1.7
+  - 1.9
```

Comment:
Is `1.9` required by gocyclo? As the agent currently is built with go  `1.7`, we probably should also update it to build with `1.9`.

Reply:
https://github.com/aws/amazon-ecs-agent/issues/681 has some of the work necessary for us to upgrade.  Beyond that, 72056b389af566c0ae2bec675cde85369cc004dd was the last time that we upgraded and shows a bit more of the actual change.

Reply:
This was me testing if the later go version will work with my use of "go list" command in makefile. It seems like it's a quotation using " or ' that's causing the problem, I'm going to revert this back to 1.7 and fix the command.